CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ Our versioning strategy is as follows:
 
 * `[templates/nextjs]` `[templates/react]` `[templates/angular]` `[templates/vue]` Fixed an issue when environment variable is undefined (not present in .env), that produced an "undefined" value in generated config file ([#1875](https://github.com/Sitecore/jss/pull/1875))
 * `[templates/nextjs]` Fix embedded personalization not rendering correctly after navigation through router links. ([#1911](https://github.com/Sitecore/jss/pull/1911))
+* `[sitecore-jss-nextjs]` The redirects middleware issue has been fixed for the Netlify service. ([#1899](https://github.com/Sitecore/jss/pull/1899))
```

Comment:
Move it, please, under JSS 22.1.2 (new section), since it will be a part of hotfix